### PR TITLE
PWN-4052 - Add UI KIT module

### PR DESCRIPTION
**Pull request by davran1001**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-4052

Add submodule of UI KIT module
Screen Shot 2022-07-20 at 19 13 10
Screen Shot 2022-07-20 at 19 13 22
Screen Shot 2022-07-20 at 19 13 43

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -201,6 +207,7 @@ android {
 dependencies {
     implementation project(':solanaj')
     implementation project(':ui-kit')
+    implementation project(':ui-kit-sample')
```
Comment (kabramovp2p) [inline app/build.gradle:210]: it seems to me that app have all dependencies
can you check size of app with this impl and without ?

i do not like it if we will have more then we need in our app like size (classes / resources/ etc)

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -50,6 +51,21 @@
 
         </activity>
 
+        <activity
```
Comment (kabramovp2p) [inline app/src/main/AndroidManifest.xml:54]: yes and same here
if we will install this app there will be 2 icons for run app and sample
as far as i see

File: app/build.gradle (hunk 1 above)
Comment (davran1001) [reply app/build.gradle:210]: 33 MB vs 32 MB

**Comment by gslevinkov** [comment]
fix the CircleCI please

File: settings.gradle [hunk 3]
```diff
@@ -1,4 +1,19 @@
 include ':solanaj'
 include ':app'
-include ':ui-kit'
-rootProject.name = "P2P Wallet"
\ No newline at end of file
+rootProject.name = "P2P Wallet"
+addSubfolderProjects("p2p-wallet-ui-app/ui-kit")
```
Comment (kuznetsov) [inline settings.gradle:4]: I don't think the `p2p-wallet-ui-app` is a good name for that repo. It is actually ui kit repository with ui kit library itself with the sample

**Review by kuznetsov — CHANGES_REQUESTED** [review]

File: settings.gradle (hunk 3 above)
Comment (kuznetsov) [reply settings.gradle:4]: May be the `android-ui-kit`

**Pull request closed without merging**